specs/gcpresource.spec

```
@@ -0,0 +1,101 @@
+# Model
+model:
+  rest_name: gcpresource
+  resource_name: gcpresources
+  entity_name: GCPResource
+  package: pandemona
+  group: pcn/infrastructure
+  description: Represents a GCP cloud resource such as a virtual machine.
+  extends:
+  - '@identifiable-stored'
+  - '@migratable'
+  - '@namespaced'
+  - '@zoned'
+
+# Indexes
+indexes:
+- - namespace
+  - selflink
+- - namespace
+  - nativeID
+- - namespace
+  - accountid
+
+# Attributes
+attributes:
+  v1:
+  - name: RRN
+    description: Prisma Cloud RRN.
+    type: string
+    exposed: true
+    omit_empty: true
+
+  - name: accountID
+    exposed_name: accountId
+    description: Cloud account ID associated with the resource.
+    type: string
+    exposed: true
+    stored: true
+    example_value: cna-dev-123456
+    omit_empty: true
+
+  - name: data
+    description: The json encoded data that represents the resource.
```

The `.spelling` already has JSON-encoded, so why not use it?
```suggestion
    description: TheJSON-encoded data that represents the resource.
```